config/filter.d/mysqld-auth.conf

```diff
@@ -17,7 +17,7 @@ before = common.conf
 
 _daemon = mysqld
 
-failregex = ^%(__prefix_line)s(?:\d+ |\d{6} \s?\d{1,2}:\d{2}:\d{2} )?\[Warning\] Access denied for user '\w+'@'<HOST>' (to database '[^']*'|\(using password: (YES|NO)\))*\s*$
+failregex = ^%(__prefix_line)s(?:\d+ |\d{6} \s?\d{1,2}:\d{2}:\d{2} )?\[\w+\] Access denied for user '[^']+'@'<HOST>' 
 
```

Comment:
> didn't hurt anyone

still not (for example the "warning" was also not bothered until today)...
We have already time talk about the subject "trailing anchor", or not? :)

By building of regexp I follow always minimalist principle. And all standing after `Access denied for user ...<HOST>` bears no meaning for decision on ban... (is only a possible disorder for future changes, any extensions of authentication methods or still not yet occurring cases, that are unknown for us, etc.)